### ISSUE TITLE
Ability to specify the keybase.txt location

### ISSUE THREAD

**Issue by Southclaws** [keybase/keybase-issues#3127]
I've recently rewritten my homepage and I'm using a server setup that only serves out of /static, so my keybase path changed to /static/keybase.txt.

It would be great to be able to specify a path for situations like these. I could set up my reverse proxy to just swap the path out but I think the ability for users to specify their path for any hosting setup would be a neat solution.

**Comment by jrtapsell** [comment]
This could be useful for verifying additional identities. For example, RHUL gives a directory to each student, so it could be used to verify students (these take the form personal.rhul.ac.uk/<username_start>/<username_end>/, for example mine is [personal.rhul.ac.uk/zcva/821](personal.rhul.ac.uk/zcva/821)). I do not know how common this is in general, but most of the unis I have checked seem to provide it in some form.

There could be an issue around how much of the site you are verifying. If verifying one subdirectory applies to the whole domain then your static file would verify your whole site for you, but fail for my example above, as I would be verified for all of `personal.rhul.ac.uk`.

A potential workaround for your use case could be using [.well-known](https://keybase.io/docs/keybase_well_known), depending on if your server supports it.

**Comment by mitar** [comment]
Yes, I think Keybase should use `.well-known` location instead of a file at the root.

**Comment by maxtaco** [comment]
This already works

**Comment by mitar** [comment]
What works?

**Comment by maxtaco** [comment]
https://www.facebook.com/.well-known/keybase.txt

**Comment by mitar** [comment]
Is this documented somewhere?

**Comment by maxtaco** [comment]
I believe the instructions for how to post a proof show you both options
